internal/plugins/helm/v1/scaffolds/templates/role.go

```diff
@@ -201,8 +201,6 @@ var DefaultRoleScaffold = Role{
 // discovery API to lookup each resource in the resulting manifest.
 // The role scaffold will have IsClusterScoped=true if the chart lists cluster scoped resources
 func GenerateRoleScaffold(dc roleDiscoveryInterface, chart *chart.Chart) Role {
-	log.Info("Generating RBAC rules")
```

Comment:
👍 I am ok with the nits in the logs besides not be part of the scope of this one. they are very small and we IMO we can move forward with them here.